CHANGELOG.md

```diff
@@ -28,6 +31,7 @@ NOTE: We are deprecating several collectors in this release.
 * [IMPROVEMENT] Ignore autofs filesystems on linux #384
 * [IMPROVEMENT] Replace some FreeBSD collectors with pure Go versions #385
 * [IMPROVEMENT] Use filename as label, move 'label' to own metric #411 (hwmon)
+* [CHANGE] Report collector success/failure and duration per scrape. #516
```

Comment:
We always put CHANGE lines at the top. Please have a look at the prometheus/prometheus CHANGELOG and follow that style.